Source/CLI/Help.cpp

```diff
@@ -11,18 +11,53 @@
 //---------------------------------------------------------------------------
 
 //---------------------------------------------------------------------------
+#include "Common/Core.h"
 #include "Help.h"
 #include "Config.h"
 //---------------------------------------------------------------------------
 
 //***************************************************************************
 //
 //***************************************************************************
+//---------------------------------------------------------------------------
+String Program_Name(const String &Set)

```

Comment:
Maybe, but I don't see how to do differently.
BTW, tested on Windows and it always shows the default value, looks normal as there is no glbal variable so argv[0] is not "saved". I don't see the idea behind the commit.